Assets/Scripts/Input/InputManager.cs

```diff
@@ -44,7 +44,7 @@ private void Start()
         
         void IInputManager.ActivateMenuInputControl()
         {
-            this.m_PlayerInput.SwitchCurrentActionMap("Menu");
+            // this.m_PlayerInput.SwitchCurrentActionMap("Menu");
```

Comment:
Remove this, additionally properly order the fields and see if it can be contextualised